mw-webapp/src/utils/LocalStorage.ts

```diff
@@ -76,5 +78,15 @@ type LocalStorageData = {
    */
   language: Language;
 
+  /**
+   * Supported statistics visibility
+   */
+  statisticsVisibility: StatisticsVisibility;
+
```

Comment:
Just in month it turned in a mess with dozens properties. What do you think about grouping them?

Reply:
Dont know how resolve it

Reply:
Fixed